pages/linux/ksvgtopng5.md

```diff
@@ -0,0 +1,7 @@
+# ksvgtopng5
+
+> Ksvgtopng5: Convert SVG files to PNG format.
```

Comment:
There's no need to repeat the command name again in the heading.

Also, is there a "more information" link that you can add?

Reply:
No, I can't find a link to more resources.